pages/windows/comp.md

```diff
@@ -0,0 +1,32 @@
+# comp
+
+> Compare the contents of two files or sets of files.
+> Use wildcards (*) to compare sets of files.
```

Comment:
Cool! Looks ok to me. Will this be an issue for clients? GitHub's diff renderer doesn't appear to like it. Should we wrap it in `*`?

Reply:
I think it's a feature of comp. I was going to add an example, but it seemed a bit weird doing;

```powershell
comp {{path/to/*}} {{path/to/*}}
```

Or something like that.

Reply:
I don't think its that weird as long as you write the explanation above:

```markdown
- compare sets of files:

`comp {{path/to/folder/a/*}} {{path/to/folder/b/*}}`
```

Reply:
@sbrl, using a backslash seems to work well. But I'm only testing on the Node client. 🤔

With `***` it just renders all 3 of them for me.

Reply:
Ah. Let's leave it then :-)